week-2-wide-perspective/w02e02-data-manipulation/src/api/getOrders.ts

```diff
@@ -0,0 +1,10 @@
+export const getOrders = async () => {
+  try {
+    const response = await fetch('http://localhost:3000/orders')
+    if (response.ok) {
+      return await response.json()
+    }
+  } catch (e) {
+    console.error(e)
+  }
+}
```

Comment:
Mam dla Ciebie `challenge` - zmień proszę to API i zastosuj `ajax` ale z `RxJS` 

zobacz:
https://www.learnrxjs.io/learn-rxjs/operators/creation/ajax

wtedy połącz strumień z `ajax` z tym co masz w `OrdersService`

Albo zastosuj podejście rodem z `Angular` i zrób osobny AjaxService (HttpService)